.github/workflows/on_push-main.yml

```diff
@@ -7,12 +7,60 @@ on:
   workflow_dispatch:
 
 jobs:
-  onPushDevelopJob:
-    name: Verify code base when pushed
+  runSonarAnalysis:
```

Comment:
Is it right we're renaming this `runSonarAnalysis` when it's doing much more than that?

Reply:
My thinking was that: the purpose of this job is to run the analysis, the other steps (cloning, running the tests, etc.) are  all just to meet that goal, but can rename it if you think that's not clear?

Reply:
Updated it to mention running automation tests, etc.